projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/secrets/service/vault/VaultJobSecretsService.java

```diff
@@ -57,7 +64,17 @@ public void updateJobSecrets(String jobName, Map<String, Object> secrets) {
                       return entry.getValue();
                     }));
 
-    vaultJobSecrets.setSecretsJson(new JSONObject(updatedSecrets).toString());
+    String updatedSecretsString = new JSONObject(updatedSecrets).toString();
+    try {
+      if (updatedSecretsString.getBytes("UTF-8").length > sizeLimit) {
```

Comment:
Again magic word. Fortunatelh since java 8 there is [StandardCharsets.UTF_8](https://docs.oracle.com/javase/8/docs/api/java/nio/charset/StandardCharsets.html#UTF_8)

Reply:
fixed.